### PR TITLE
Include multiple `purescript` org repos as updated libraries

### PR DESCRIPTION
**Pull request by JordanMartinez**

### CODE REVIEW

File: src/groups/purescript.dhall [hunk 1]
```diff
@@ -97,7 +97,7 @@
 , effect =
   { dependencies = [ "prelude" ]
   , repo = "https://github.com/purescript/purescript-effect.git"
-  , version = "v2.0.1"
+  , version = "master"
```
Comment (andys8) [inline src/groups/purescript.dhall:100]: Is this intended?

Comment (JordanMartinez) [reply src/groups/purescript.dhall:100]: It was, but I'm wondering whether the original approach I was using to update the ecosystem should still be done.

Comment (andys8) [reply src/groups/purescript.dhall:100]: Comparing it with prelude, I do now understand the intention.

Comment (JordanMartinez) [reply src/groups/purescript.dhall:100]: I think we'll use `bower` to update each core library and then use `spago` to verify that previous PRs made to core libraries still work on later PureScript rc releases. For example, `prelude` was updated when `v0.14.0-rc2` was the release candidate. Harry made the `-rc3` version now, so does `prelude` still compile on it? We could use `spago` and this branch to track that so we don't need to waste time updating each individual core library whenever a new release candidate is made.

**Pull request merged**